environment.Makefile

```diff
@@ -54,6 +54,11 @@ ifeq ($(UNAME_ARCH), arm64)
 	rm -fr build/$(PROJECT_NAME)
 endif
 
+.PHONY: jupyter
+jupyter: install ## Start a jupyter notebook with editable layer package
+	@pip install -e .
```

Comment:
When you do `poetry run` that should already have the package installed in editable mode. Are you sure we need `pip install`? Will this even be the correct pip if poetry is using a virtual env?

Reply:
During tests was using MacOS and `conda`, was installing via pip to conda venv and then running jupyter directly instead of via poetry. After testing and making it work in Linux that `pip` is a leftover and not needed. Thanks!